entregables/site/Docs/Manual-de-Usuario/Evaluaciones.markdown

```diff
@@ -0,0 +1 @@
+# Evaluaciones
```

Comment:
I would add a //TODO here, or something that is searchable so that it is easy later to find out everything that's missing to be completed.